Frameworks/CoreText/CTFrame.mm

```diff
@@ -78,8 +79,7 @@ CFRange CTFrameGetVisibleStringRange(CTFrameRef frame) {
  @Notes
 */
 CGPathRef CTFrameGetPath(CTFrameRef frame) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return frame ? static_cast<_CTFrame*>(frame)->_path.get() : nil;
```

Comment:
outside side effects?

Also Macros make much more sense here and given it's defacto standard, it would be nice to have all of the code in a certain way.

---
In reply to: [84375730](https://github.com/Microsoft/WinObjC/pull/1197#discussion_r84375730) [](ancestors = 84375730)